### PR TITLE
Follow up summary view

### PR DESCRIPTION
**Pull request by denektenina**

### CODE REVIEW

File: frontend/src/views/Evaluation/FollowUp/Summaries/Bowtie.tsx [hunk 1]
```diff
@@ -110,23 +110,23 @@ const Text = styled(Typography)`
 `
 
 interface Props {
-    answersWithBarrier: AnswersWithBarrier[]
+    answersWithBarrier: BarriersWithAnswers[]
```
Comment (maiaht) [inline frontend/src/views/Evaluation/FollowUp/Summaries/Bowtie.tsx:113]: Maybe the prop name should be in the same order as the name of the interface, such that `answersWithBarrier` -> `barrierWithAnswers`?

File: frontend/src/views/Evaluation/FollowUp/Summaries/BowtieView.tsx [hunk 2]
```diff
@@ -17,18 +17,18 @@ const ScrollableDiv = styled.div`
...
+const BowtieView = ({ barriersWithProgressionAnswer }: Props) => {
     return (
         <>
             <Box p="20px">
                 <Typography variant="h2" style={{ marginBottom: '30px' }}>
                     Bowtie model
                 </Typography>
                 <ScrollableDiv>
-                    <Bowtie answersWithBarrier={answersWithBarrier} />
+                    <Bowtie answersWithBarrier={barriersWithProgressionAnswer} />
```
Comment (maiaht) [inline frontend/src/views/Evaluation/FollowUp/Summaries/BowtieView.tsx:31]: `answersWithBarrier` -> `barrierWithAnswers` here as well?

File: frontend/src/views/Evaluation/FollowUp/Summaries/Bowtie.tsx (hunk 1 above)
Comment (denektenina) [reply frontend/src/views/Evaluation/FollowUp/Summaries/Bowtie.tsx:113]: Yes, that actually was the old name. Which is probably why it is still the name in the interface (?) the reason I changed it is that when it's created we go through a list of barriers and assign answers to it, so it felt more correct to highlight the barrier-part of it, but I'm not very attached to this change so I can change it back

File: frontend/src/views/Evaluation/FollowUp/Summaries/BowtieView.tsx (hunk 2 above)
Comment (denektenina) [reply frontend/src/views/Evaluation/FollowUp/Summaries/BowtieView.tsx:31]: I can change the order, but I want to keep the word progression in there to show better what it is

**Review by maiaht — APPROVED** [review]

**Pull request merged**